src/gamma.jl

```diff
@@ -56,6 +57,12 @@ function _digamma(x::BigFloat)
     return z
 end
 
+"""
+    _cotpi(x) = cot(π * x)
+
+Accurate for integer arguments
+"""
+_cotpi(x) = cospi(x) / sinpi(x)
```

Comment:
On Julia 1.10, we can use `tanpi`:
```suggestion
if VERSION >= v"1.10.0-DEV.525"
    _cotpi(x) = inv(tanpi(x))
else
    _cotpi(x) = cospi(x) / sinpi(x)
end
```

Reply:
will do!